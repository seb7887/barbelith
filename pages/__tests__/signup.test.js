/* eslint-env jest */
import 'react-testing-library/cleanup-after-each';

import { render } from 'react-testing-library';
import SignupPage from '../signup';

describe('<SignupPage />', () => {
  it('renders and matches snapshot', () => {
    const { asFragment } = render(
      <SignupPage />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
