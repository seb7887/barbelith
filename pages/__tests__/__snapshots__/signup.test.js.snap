// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignupPage /> renders and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Signup-root-1"
    data-testid="signup-form"
  >
    <div
      class="MuiPaper-root-9 MuiPaper-elevation2-13 MuiPaper-rounded-10 Signup-paper-2"
    >
      <div
        class="MuiAvatar-root-36 MuiAvatar-colorDefault-37 Signup-avatar-4"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root-39"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <defs>
            <path
              d="M0 0h24v24H0V0z"
              id="a"
            />
          </defs>
          <path
            d="M1 21h12v2H1zM5.245 8.07l2.83-2.827 14.14 14.142-2.828 2.828zM12.317 1l5.657 5.656-2.83 2.83-5.654-5.66zM3.825 9.485l5.657 5.657-2.828 2.828-5.657-5.657z"
          />
        </svg>
      </div>
      <h1
        class="MuiTypography-root-48 MuiTypography-h5-64"
      >
        Sign up
      </h1>
      <form
        class="Signup-form-5"
      >
        <div
          class="MuiFormControl-root-84 MuiFormControl-marginNormal-85 MuiFormControl-fullWidth-87"
        >
          <label
            class="MuiFormLabel-root-99 MuiFormLabel-required-104 MuiInputLabel-required-92 MuiInputLabel-root-88 MuiInputLabel-formControl-93 MuiInputLabel-animated-96"
            data-shrink="false"
            for="name"
          >
            Name
            <span
              class="MuiFormLabel-asterisk-105"
            >
               *
            </span>
          </label>
          <div
            class="MuiInputBase-root-119 MuiInput-root-106 MuiInput-underline-110 MuiInputBase-formControl-120 MuiInput-formControl-107"
          >
            <input
              aria-invalid="false"
              class="MuiInputBase-input-129 MuiInput-input-114"
              id="name"
              name="name"
              required=""
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          class="MuiFormControl-root-84 MuiFormControl-marginNormal-85 MuiFormControl-fullWidth-87"
        >
          <label
            class="MuiFormLabel-root-99 MuiFormLabel-required-104 MuiInputLabel-required-92 MuiInputLabel-root-88 MuiInputLabel-formControl-93 MuiInputLabel-animated-96"
            data-shrink="false"
            for="email"
          >
            Email
            <span
              class="MuiFormLabel-asterisk-105"
            >
               *
            </span>
          </label>
          <div
            class="MuiInputBase-root-119 MuiInput-root-106 MuiInput-underline-110 MuiInputBase-formControl-120 MuiInput-formControl-107"
          >
            <input
              aria-invalid="false"
              class="MuiInputBase-input-129 MuiInput-input-114 MuiInputBase-inputType-132 MuiInput-inputType-117"
              id="email"
              name="email"
              required=""
              type="email"
              value=""
            />
          </div>
        </div>
        <div
          class="MuiFormControl-root-84 MuiFormControl-marginNormal-85 MuiFormControl-fullWidth-87"
        >
          <label
            class="MuiFormLabel-root-99 MuiFormLabel-required-104 MuiInputLabel-required-92 MuiInputLabel-root-88 MuiInputLabel-formControl-93 MuiInputLabel-animated-96"
            data-shrink="false"
            for="password"
          >
            Password
            <span
              class="MuiFormLabel-asterisk-105"
            >
               *
            </span>
          </label>
          <div
            class="MuiInputBase-root-119 MuiInput-root-106 MuiInput-underline-110 MuiInputBase-formControl-120 MuiInput-formControl-107"
          >
            <input
              aria-invalid="false"
              class="MuiInputBase-input-129 MuiInput-input-114 MuiInputBase-inputType-132 MuiInput-inputType-117"
              id="password"
              name="password"
              required=""
              type="password"
              value=""
            />
          </div>
        </div>
        <button
          class="MuiButtonBase-root-162 MuiButton-root-136 MuiButton-contained-147 MuiButton-containedSecondary-149 MuiButton-raised-150 MuiButton-raisedSecondary-152 MuiButton-fullWidth-161 Signup-submit-6"
          data-testid="submit"
          tabindex="0"
          type="submit"
        >
          <span
            class="MuiButton-label-137"
          >
            Sign up
          </span>
          <span
            class="MuiTouchRipple-root-181"
          />
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
